src/components/Button/Button.stories.tsx

```diff
@@ -101,6 +101,57 @@ export const DangerGhost: Story = {
   },
 }
 
+export const PrimaryLink: Story = {
```

Comment:
Is the code snippet from these stories correctly rendered?

Reply:
I have inserted some screenshot on the PR description.

Reply:
not about the code snippets 😅

Reply:
Ah okok 

![Schermata del 2024-07-23 10-30-17](https://github.com/user-attachments/assets/2766615e-734c-4f13-b3da-fe72fb392c16)